src/components/simulator/SimulatorSummaryTable.tsx

```diff
@@ -35,19 +35,15 @@ export const SimulatorSummaryTable = ({
   const quoteSymbol = quoteToken.symbol;
 
   return (
-    <article className="grid grid-cols-[auto,auto] grid-rows-4 items-center justify-evenly gap-2 md:grid-cols-[auto,auto,auto,auto] md:grid-rows-2">
-      <Arrow className="mx-6 h-16 w-16 text-green" />
-      <span>
-        {buyMin}-{buyMax} {baseSymbol} per {quoteSymbol}
-      </span>
-      <span className="mx-4 text-white/40">|</span>
+    <table className="grid grid-cols-[auto,auto] grid-rows-4 items-center justify-evenly gap-6 md:grid-cols-[auto,auto,auto,auto] md:grid-rows-2">
```

Comment:
I think `<table>` is the wrong element for this, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#technical_summary